README.md

```diff
@@ -136,6 +136,11 @@ or
 
     ./gradlew cli:run --args="--info analyze -f JSON -i /project -o /project/ort/analyzer"
 
+## Running on CI
+
+A basic ORT pipeline (using the analyzer, scanner and reporter) can be easily run on [Jenkins CI](https://jenkins.io/)
+by using the [Jenkinsfile](./Jenkinsfile) in a (declarative) pipeline job.
```

Comment:
You could add a link to the pipeline docs: https://jenkins.io/doc/book/pipeline/

Reply:
Done.